<launch>
    <!-- ROS Nodelet manager -->
    <node pkg="nodelet" type="nodelet" name="BLUEROV2_STATES_master" args="manager" output="screen">
            <!-- <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/alan_pose_estimation.yaml" /> -->
    </node> 

    <!-- ArucoNodelet -->
    <node pkg="nodelet" type="nodelet" name="ImuDo" args="load BLUEROV2_STATES/ImuDoNodelet BLUEROV2_STATES_master" output="screen">
        <!-- <rosparam command="load" file="$(find alan_state_estimation)/launch/configs/ARUCO_SUBPUB_topics.yaml" /> -->
    </node>

</launch>